### PR TITLE
PRM-1398 Added Appeal enums to backend models

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

File: test/models/v3/getPenaltyDetails/appealInfo/AppealLevelEnumTest.scala [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.v3.getPenaltyDetails.appealInfo
+
+import base.SpecBase
+import play.api.libs.json.{JsString, Json}
+
+class AppealLevelEnumTest extends SpecBase {
```
Comment (LewisMitchell) [inline test/models/v3/getPenaltyDetails/appealInfo/AppealLevelEnumTest.scala:22]: Could you rename this to:
```suggestion
class AppealLevelEnumSpec extends SpecBase {
```
and the file name

File: test/models/auditing/v2/UserHasPenaltyAuditModelSpec.scala [hunk 2]
```diff
@@ -163,11 +164,7 @@ class UserHasPenaltyAuditModelSpec extends SpecBase with LogCapturing {
           lateSubmissions = None,
           expiryReason = None,
           appealInformation = Some(Seq(
-            AppealInformation(
-              appealStatus = Some("A"),
-              appealLevel = Some("01")
-            ))
-          ),
+            AppealInformationType(appealStatus = Some(AppealStatusEnum.Under_Appeal), appealLevel = Some(AppealLevelEnum.HMRC)))          ),
```
Comment (LewisMitchell) [inline test/models/auditing/v2/UserHasPenaltyAuditModelSpec.scala:167]: Could you remove the padded space

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: test/models/v3/getPenaltyDetails/appealInfo/AppealLevelEnumTest.scala (hunk 1 above)
Comment (AndrewRButcher) [reply test/models/v3/getPenaltyDetails/appealInfo/AppealLevelEnumTest.scala]: Done ✅

File: test/models/auditing/v2/UserHasPenaltyAuditModelSpec.scala (hunk 2 above)
Comment (AndrewRButcher) [reply test/models/auditing/v2/UserHasPenaltyAuditModelSpec.scala]: Done ✅

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**